src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -137,24 +139,24 @@ The Hero currently merges three times a day: At 9 AM (morning), 12 PM (noon), an
 
 To start a release:
 
-1. Create a branch based off develop Github Desktop by clicking **Current Branch** in the top header, clicking **New Branch** in the dropdown, and then selecting **Develop**.
-2. Name the branch following this pattern: `daily-release/mm-dd-yy-morning/noon/evening`. Here's an example: `daily-release/10-27-21-morning`.
-3. Push your changes by clicking **Push Origin** in GitHub Desktop.
+1. Create a branch of `develop` using Github Desktop by clicking **Current Branch**. in the top header, clicking **New Branch** in the dropdown, and then selecting **Develop**.
+2. Name the branch with this pattern: `daily-release/mm-dd-yy-morning/noon/evening`. For example, a daily release happening at 9am on October 27, 2021 would follow this style: `daily-release/10-27-21-morning`.
+3. Push your changes with **Push Origin** in GitHub Desktop.
```

Comment:
I prefer slightly: "To push your changes, in GitHub Desktop click **Push Origin**."